index.js

```diff
@@ -23,3 +23,154 @@ menuItems.forEach(
     doesntExist.addEventListener('click', openMenuHam);
   },
 );
+const projects = [
+  {
+    name: 'Multi-Post Stories',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    image: './css/Works/preview.png',
```

Comment:
- [ ] Pop-up window is looking great! Just one thing, there is a typo in the image path, let's fix it to display your images properly.